contributors/design-proposals/storage/pv-monitoring-proposal.md

```diff
@@ -0,0 +1,127 @@
+# PV monitoring proposal
+
+Status: Pending
+
+Version: Alpha
+
+Implementation Owner: NickrenREN@ 
+
+## Motivation
+
+For now, kubernetes has no way to monitor the PVs, which may cause serious problems. 
+For example: if volumes are unhealthy, and pods do not know that and still try to get and write data, 
+which will lead to data loss and unavailability of services. 
+So it is necessary to have a mechanism for monitoring PVs and react when PVs have problems.
+
+## Proposal
+
+We can separate the proposal into two parts:
+
+* monitoring PVs and marking them if they have problems
```

Comment:
What exactly will it monitor btw? Are you talking about bad blocks or fsck? Please add some examples.

Reply:
yeah, sth like that,  e.g.
For local PV: its Path( directory) is deleted by mistake , node failure...

Reply:
It would be good to explicitly write down some specific error conditions somewhere.